conf/application.conf

```diff
@@ -168,6 +168,7 @@ microservice {
       enrolment-store = true
       fx-enabled = true
       public-message-banner = true
+      accessibility-statement = true
```

Comment:
We will need to add this to app-config-base

Reply:
added